Enable access to the low-level API from the `Account` (#522)
iotaledger/identity.rs: examples/README.md

| 5 | [account_lazy](./account/lazy.rs) | How to take control over publishing DID updates manually, instead of the default automated behavior. |
| 6 | [account_signing](./account/signing.rs) | Using a DID to sign arbitrary statements and validating them. |
| 7 | [account_multiple](./account/multiple_identities.rs) | How to create multiple identities from a builder and how to load existing identities into an account. |

> | 8 | [account_unchecked](./account/unchecked.rs) | Making an update to the custom properties of a document, by using the account's unchecked methods. |

Reviewer: I'm not sure what phrasing we're supposed to use for example descriptions, it's pretty inconsistent.

```suggestion
| 8 | [account_unchecked](./account/unchecked.rs) | How to update the custom properties of a DID document directly by using the account's unchecked methods. |
```
